src/seeders/seed.ts

```diff
@@ -148,13 +105,76 @@ export const createEvaluationResults = async () => {
   })
 }
 
+/**
+ * Production
+ */
+
+const createRoles = async () => {
+  const roles = [
+    {
+      name: "khv2_hr_evaluators",
+      user_id: 15,
+    },
+    {
+      name: "khv2_hr_evaluators",
+      user_id: 86,
+    },
+    {
+      name: "khv2_hr_evaluators",
+      user_id: 20032,
+    },
+    {
+      name: "khv2_hr_evaluators",
+      user_id: 20090,
+    },
+  ]
+  for (const role of roles) {
+    const existingRole = await prisma.user_roles.findFirst({
+      where: {
+        name: role.name,
+        user_id: role.user_id,
+      },
+    })
+    if (existingRole === null) {
+      await prisma.user_roles.create({
+        data: role,
+      })
+    }
+  }
+}
+
+const createEmailTemplates = async () => {
+  const emailTemplate = await prisma.email_templates.findFirst({
+    where: {
+      name: "Create Evaluation Administration Template",
+    },
+  })
+  if (emailTemplate === null) {
+    await prisma.email_templates.create({
+      data: [
+        {
+          name: "Create Evaluation Administration Template",
+          template_type: "Create Evaluation",
+          is_default: true,
+          subject: "Request for Evaluation",
+          content: `Dear Evaluator,\n\nWe kindly request your participation in the evaluating the following during this period: {{evaluation_period}}. Please follow this {{link}} to access {{evaluation_name}}.\n\nThe deadline for completing these evaluations is on {{eval_schedule_end_date}}. If you encounter any technical issues or have questions regarding the process, please feel free to reach out to any of the HR team members.\n\nThanks and Best Regards,\nKH360 Team`,
```

Comment:
Grammatical error?

Reply:
the content? 🤔 @emmaacha

Reply:
Yes po kuys @jatnerubia this part: `"We kindly request your participation in the evaluating the following..."`. I think it should either be `"in evaluating the following"` or `"in the evaluation of the following"`

Reply:
ohh, ask shei about this

Reply:
This is the final content kuys @jatnerubia according to ate Shei 😁
```
Dear Evaluator,

We kindly request your participation in evaluating the following during this period: {{evaluation_period}}.
Please follow this {{link}} to access {{evaluation_name}}.

The deadline for completing these evaluations is on {{eval_schedule_end_date}}.
If you encounter any technical issues or have questions regarding the process, please feel free to reach out to any of the HR team members.

Thanks and Best Regards,
KH360 Team
```